andela/AndelaEatsUI
# #160237268 Implement delete vendor UI and functionality

**What does the PR do?**
- Implements `Delete Vendor` functionality

**Description of Task to be completed?**
- Ensures an admin can delete a vendor

**How should this be manually tested?**
- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git) and
-  cd into AndelaEatsUI and checkout to this branch `ft-implement-delete-vendor-160237268`
- run `npm run bundle` and `npm run dev`
- Log in as an `Admin`, navigate to `admin dashboard`, select `Vendors`, and click on `Delete`
- run `npm test` to ensure all tests are passing

**What are the relevant pivotal tracker stories?**
- [#160237268](https://www.pivotaltracker.com/story/show/160237268)

**Screenshots**
<img width="1680" alt="screen shot 2018-09-14 at 10 02 28 am" src="https://user-images.githubusercontent.com/26305570/45542375-980ceb00-b809-11e8-8475-641df0707779.png">
